.travis.yml

```diff
@@ -73,6 +79,7 @@ addons:
       - rsync
       - xorg-dev
       - valgrind
+      - cppcheck
```

Comment:
I think you have to update `install_osx.sh` so that `brew` installs `cppcheck`.

Reply:
no need to run cppcheck on more than one instance, so i just run it on one linux box